### DISCUSSION TITLE
how to run a batch of a sequence PLY files

### DISCUSSION THREAD

**Discussion by mickatt** [cnr-isti-vclab/PyMeshLab#199] [Q&A]
Hi

I have several dozen animation sequences in point cloud in PLY format that I would like to convert into PLY mesh sequence or obj sequence
I converted one PLY file by applying the filters but I don't know what the python command is for an entire sequence
Here is my python script for one conversion file :

> import pymeshlab as ml
ms = ml.MeshSet()
ms.load_new_mesh('test_new_0000.ply')
ms.load_filter_script('normal_surface.mlx')
ms.apply_filter_script()
ms.save_current_mesh('output_.obj')

Could you help me please?
Thanks a lot
Michael Attia

**Comment by alemuntoni** [comment]
I'm not sure I understand the question correctly, but it seems to me that you need to look how to [loop in a list of files in python](https://stackoverflow.com/questions/3207219/how-do-i-list-all-files-of-a-directory), and then apply your filters on each of them.

**Reply by mickatt** [reply]
Thanks a lot Alesssndro for your answer
Sorry of I m’not clear 
What I want is to apply the filters to a sequence PLY files 
I will check your link 
Regards
